### ISSUE TITLE
Add Recycle Bin

### ISSUE THREAD

**Issue by Github-com-foss** [beemdevelopment/Aegis#624]
Pls Add Recycle Bin, if accidentally delete the key.

**Comment by alexbakker** [comment]
I think this is a good idea. Perhaps we can misuse the group feature and add a special "Recycle bin" group that entries get assigned to, instead of actually deleting them. Only if an entry that is already in the "Recycle bin" group is deleted, we actually delete it.

**Comment by lynn-stephenson** [comment]
@alexbakker I am taking a guess you would treat the "Recycle Bin" group to be visually hidden?

**Comment by alexbakker** [comment]
@lynn-stephenson Yes. It may be a good idea to have a separate way to access the Recycle bin though, as currently the group filter is a bit hard to find.

**Comment by OmlineEditor** [comment]
Good idea. +1

**Comment by alexbakker** [comment]
There's another issue that suggests adding an "Archive" option: #1108. That's probably a better name for the intent here.

**Comment by alexbakker** [comment]
We don't do ETA's

**Comment by noperator** [comment]
Do you do ETAs _now_? (I'd love this feature!)
